storio-content-resolver/src/main/java/com/pushtorefresh/storio/contentresolver/queries/DeleteQuery.java

```diff
@@ -71,6 +72,17 @@ public String where() {
         return whereArgs;
     }
 
+    /**
+     * Returns the new builder that has the same contents as this query.
+     * It can be updated and create new queries.
```

Comment:
"It can be used to create new queries", it's not correct to say "and create" in this case :)